src/main/kotlin/com/openlattice/chronicle/storage/ChroniclePostgresTables.kt

```diff
@@ -124,6 +126,18 @@ class ChroniclePostgresTables {
             )
             .primaryKey(STUDY_ID, DEVICE_ID) //Just in case device is used across multiple studies
 
+        @JvmField
+        val TIME_USE_DIARY_SUBMISSION = PostgresTableDefinition("time_use_diary_submissions")
+            .addColumns(
+                TUD_ID,
+                ORGANIZATION_ID,
+                STUDY_ID,
+                PARTICIPANT_ID,
+                SUBMISSION_DATE,
+                SUBMISSION
+            )
+            .primaryKey(TUD_ID)
```

Comment:
Does each submission get its own id or is this the id of a specific tud. I ask because if it the latter then I don't think this primary key is sufficient.

Reply:
Each submission gets its own ID, which means TUD_ID is a misnomer. Will rename to SUBMISSION_ID to make it more intuitive.